gammapy/analysis/config.py

```diff
@@ -67,6 +67,8 @@ class FrameEnum(str, Enum):
 
 class BackgroundMethodEnum(str, Enum):
     reflected = "reflected"
+    fov = "fov_background"
+    no_background = "None"
```

Comment:
Since all fields should be `str` types, I'd suggest to use `no_background = ""` instead